src/skin/tooltips.cpp

```diff
@@ -597,12 +598,19 @@ void Tooltips::addStandardTooltips() {
             << tr("Eject")
             << tr("Ejects track from the player.");
 
-    add("hotcue")
-            << tr("Hotcue")
-            << QString("%1: %2").arg(leftClick, tr("If hotcue is set, jumps to the hotcue."))
-            << tr("If hotcue is not set, sets the hotcue to the current play position.")
-            << quantizeSnap
-            << QString("%1: %2").arg(rightClick, tr("Opens a menu to clear hotcues or edit their labels and colors."));
+    add("hotcue") << tr("Hotcue")
+                  << QString("%1: %2").arg(leftClick,
+                             tr("If hotcue is set, jumps to the hotcue."))
+                  << tr("If hotcue is not set, sets the hotcue to the current "
+                        "play position.")
+                  << quantizeSnap
+                  << QString("%1: %2").arg(rightClick,
+                             tr("Opens a menu to clear hotcues or edit their "
+                                "labels and colors."))
+                  << QString("%1 + %2: %3")
+                             .arg(rightClick,
+                                     shift,
+                                     tr("Delete selected cuepoint."));
```

Comment:
```suggestion
                                     tr("Delete selected hotcue."));
```

Reply:
mmhh, when we have saved loops, this could also be a cue loop right ?

Reply:
I renamed it for now